lib/ws_servers/api/open_auth_bitfinex_connection.js

```diff
@@ -56,9 +57,23 @@ module.exports = async (conf) => {
       case 'pu': return send(ws, ['data.position', 'bitfinex', msgData])
       case 'pc': return send(ws, ['data.position.close', 'bitfinex', msgData])
       case 'os': return send(ws, ['data.orders', 'bitfinex', msgData])
-      case 'on':
+      case 'on': {
+        const order = receiveOrder(msgData)
+
+        notifyOrderSubmitted(ws, 'bitfinex', order)
+        return send(ws, ['data.order', 'bitfinex', msgData])
+      }
       case 'ou': return send(ws, ['data.order', 'bitfinex', msgData])
```

Comment:
may be we could add one case for `partially filled` case like `notifyPartialOrderFilled`? And later when we find more cases, we can add accordingly later..

Reply:
it was not part of the requirement. I will ask Ivan about it

Reply:
@avsek477 he replied that he would create a separate task for partial filling

Reply:
suree ok